docs/manual.rst

```diff
@@ -1861,6 +1861,30 @@ The gallery pages are generated using the ``gallery.tmpl`` template, and you can
 customize it there (you could switch to another lightbox instead of baguetteBox, change
 its settings, change the layout, etc.).
 
+Images in galleries may be provided with captions and given a specific
+ordering, by creating a file in the gallery directory called metadata.yml.
+This YAML file should contain a 'name' field for each image in the gallery,
+and (if desired) either or both of 'caption' and 'order'. Only one
+`metadata.yml` is needed per gallery. Here is an example, showing names,
+captions and ordering.
+
+.. code:: yaml
+
+    ---
+    name: ready-for-the-acid-wash.jpg
```

Comment:
Is this necessary? I would prefer something where users can specify only the files they want changed from the default.

Reply:
I've updated the docs to make it plain that you only need to list images for which a specific caption and/or order is required.